worlds/stardew_valley/regions.py

```diff
@@ -353,8 +354,81 @@ def inverted(self):
 ]
 
 
+# Attempt to implement remote loading of modded data if relevant (to avoid circular references and clutter)
+def loaded_mod_region_data(mod: str, data_type: str, world_options: StardewOptions):
```

Comment:
I strongly dislike using this one method for 4 completely different and independent types of operations.
My understanding is that the data_type parameter decides if you're loading all the mod regions, all the mod connections, removing regions or removing connections. These should be 4 methods instead, and their names could reflect what they do